### ISSUE TITLE
crashed necrobot!

### ISSUE THREAD

**Issue by eru0921** [NECROBOTIO/NecroBot#4147]
It has been plagued by this error . We move the 8bot. what is the reason?

screen shot.

http://imgur.com/a/uoabx

My config.
http://pastebin.com/sNVQ0FmW

**Comment by Andrerm124** [comment]
By the looks of things it could be something to do with your system language using unsupported characters. A recent change I've made could POSSIBLY have fixed your problem: #4116

**Comment by eru0921** [comment]
Do you want to change it in any way ? .cs File is missing

**Comment by Andrerm124** [comment]
No, hopefully your problem should be fixed when the next version is released

**Comment by eru0921** [comment]
execrash did it cause garbled of POKESTOP?

**Comment by schwartzadev** [comment]
Is this problem still happening?

**Comment by eru0921** [comment]
@schwartzadev 

Hello. Yes  It has also been plagued with this error now .
